docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1845,8 +1863,10 @@ async function getStatsForVideoPane(meeting, videoPane) {
 
   // there is no public API in WCME to get the transceiver of a receive slot, but
   // this is javascript so we can access private fields to get it anyway until we have some API for this
-  // todo: 'VIDEO-MAIN' will need to change when we support screen sharing (SPARK-377812)
-  const transceiver = multistreamConnection.recvTransceivers.get('VIDEO-MAIN')?.find((t) => t.receiveSlot === wcmeReceiveSlot);
+  const transceiver = [
+    ...(multistreamConnection.recvTransceivers.get('VIDEO-MAIN') || []),
+    ...(multistreamConnection.recvTransceivers.get('VIDEO-SLIDES') || []),
```

Comment:
the `|| []` bit is needed because WCME returns undefined for VIDEO-SLIDES for now